packages/maps/package.json

```diff
@@ -25,11 +25,12 @@
 		"emotion": "^9.0.0",
 		"emotion-theming": "^9.0.0",
 		"leaflet": "^1.3.4",
+		"ngeohash": "^0.6.3",
 		"prop-types": "^15.6.0",
 		"react-emotion": "^9.0.0",
 		"react-google-maps": "^9.4.5",
-		"rheostat": "^2.1.1",
-		"react-leaflet": "^2.1.3"
+		"react-leaflet": "^2.1.3",
+		"rheostat": "^2.1.1"
```

Comment:
@lakhansamani Are line number 32, 33 changes intentional?

Reply:
Nope it got updated with yarn install